stories/topics/proc-aap-gcp-backup-and-restore.adoc

```diff
@@ -0,0 +1,122 @@
+[id="proc-aap-gcp-backup-and-restore"]
+
+= Ansible Automation Platform Backup and Restore Process
+
+In order to backup and restore your AAP deployment, it is vital to have your existing AAP Admin Secret name and value recorded somewhere safe. In addition to this, it is also extremely important that regular manual backups of the Cloud SQL DB instance and filestore backups are taken to ensure a deployment can be restored as close as possible to its previous working state. The playbooks backup and restore provides backup and restore support for the AAP on GCP foundation deployment. 
+
+**Note that the restore process deploys a new AAP deployment with the filestore and SQL DB instance being restored to the specified backup.**
```

Comment:
You don't need the asterisks.

Reply:
want to put the statement in bold so added astericks

Reply:
OK

AAP -> {PlatformNameShort}

DB -> database